pkg/steps/multi_stage.go

```diff
@@ -420,7 +459,7 @@ func (s *multiStageTestStep) generatePods(steps []api.LiteralTestStep, env []cor
 		// the grace period for the Pod to be just larger than the grace period
 		// for the process, assuming an 80/20 distribution of work.
 		terminationGracePeriodSeconds := p(int64(gracePeriod.Seconds() * 5 / 4))
-		pod, err := generateBasePod(s.jobSpec, name, multiStageTestStepContainerName, []string{"/bin/bash", "-c", CommandPrefix + step.Commands}, image, resources, artifactDir, s.jobSpec.DecorationConfig, s.jobSpec.RawSpec(), secretVolumeMounts)
+		pod, err := generateBasePod(s.jobSpec, name, multiStageTestStepContainerName, []string{"/bin/sh", "-c", CommandExec}, image, resources, artifactDir, s.jobSpec.DecorationConfig, s.jobSpec.RawSpec(), secretVolumeMounts)
```

Comment:
Why are we using a shell to execute this? Shouldn't we just ensure that the files have a shebang and are executable?

Reply:
If a user injects the CLI into their test, we need to export a new path that includes the CLI_DIR. Using `/bin/sh` allows us to run the export command to update the env var before running the script: https://github.com/openshift/ci-tools/pull/1776/files#diff-da42b74c690675134a34fba17070b2705bdf546b16b6e84e06ef9440f637a166R687-R688

Reply:
Can we move that to the `entrypoint-wrapper` like I did for `$HOME` handling (in a separate PR, first)? The less weird Bash mangling we have the better.